src/lib/components/ui/RecentCategoryArticles.svelte

```
@@ -0,0 +1,119 @@
+<script lang="ts">
+    import type { Article, ArticleMetadata } from '$lib/types/article';
+    import { getArticles } from '$lib/stores/articles.svelte';
+    import { onMount } from 'svelte';
+    import { ArrowRight } from 'lucide-svelte';
+    import Badge from './badge/badge.svelte';
+    
+    const { 
+        maxCategories = 7,
```

ESLint

[error] 'maxCategories' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)